examples/beam/without_job_server/beam_wordcount_java.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: beam-wordcount-java
+spec:
+  template:
+    metadata:
+      labels:
+        app: beam-wordcount-java
+    spec:
+      containers:
+        - name: beam-wordcount-java
+          image: apache/beam_java_sdk:2.23.0
```

Comment:
We should use 2.22.0 which is the same as that of the flink session cluster in this example.